src/search/QuickSearchField.js

```diff
@@ -91,6 +91,9 @@ define(function (require, exports, module) {
 
         $input.on("input", this._handleInput);
         $input.on("keydown", this._handleKeyDown);
+        
+        // For search History this flag is set to false
```

Comment:
The flag is set to `null` for search History. As it's a numeric value, `false` doesn't make sense.